<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeepSeek API Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modelComboBox">
        <item>
         <property name="text">
          <string>deepseek-chat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>deepseek-coder</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="apiKeyButton">
        <property name="text">
         <string>Set API Key</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Prompt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="promptTextEdit">
      <property name="placeholderText">
       <string>Enter your prompt here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Response:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="responseTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Response will appear here...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>